docs/excel/show-hide-add-in.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Show or hide an Office Add-in in a shared runtime
+description: 'Learn how to programmatically hide or show the UI of an add-in while it runs continuously'
+ms.date: 03/02/2020
+localization_priority: Normal
+---
+
+# Show or hide an Office Add-in in a shared runtime (preview)
+
+Office Add-ins can include any of the following parts:
+
+- a task pane
+- a UI-less function file
+- an Excel custom function
+
+Until now, if an add-in had more than one of these parts, then each part ran in its own separate JavaScript runtime, with its own global object and global variables.
+
+Microsoft is now making it possible for add-ins with two or more parts to share a common JavaScript runtime. This new feature enables new preview APIs for hiding the task pane part of an add-in while the add-in remains running and for reopening the task pane later.
+
+> [!IMPORTANT]
```

Comment:
It might be worth adding an includes file specific to shared runtime (David might already have one?) so we don't have to update in multiple places

Reply:
Good idea. Done.